pootle/core/decorators.py

```diff
@@ -208,13 +208,16 @@ def set_project_resource(request, path_obj, dir_path, filename):
     resource_path = dir_path
     pootle_path = ctx_path + dir_path
 
-    # List of disabled TP paths
-    disabled_tps = TranslationProject.objects.disabled().filter(
+    # List of TP paths available for user
+    user_tps = TranslationProject.objects.for_user(request.user)
+    user_tps = user_tps.filter(
         project__code=path_obj.code,
     ).values_list('pootle_path', flat=True)
-    disabled_tps = list(disabled_tps)
-    disabled_tps.append('/templates/')
-    disabled_tps_regex = '^%s' % u'|'.join(disabled_tps)
+    user_tps  = list(user_tps)
+    user_tps_regex = '^%s' % u'|'.join(user_tps)
+    sql_regex = 'REGEXP'
+    if connection.vendor == 'postgresql':
+        sql_regex = '=~'
```

Comment:
IMHO it is better to just move this to the next `if` two lines below, so we have just one `if`.